chia/wallet/wallet_transaction_store.py

```diff
@@ -138,6 +145,8 @@ async def add_transaction_record(self, record: TransactionRecord) -> None:
             await conn.execute_insert(
                 "INSERT OR REPLACE INTO tx_times VALUES (?, ?)", (record.name, bytes(record.valid_times))
             )
+            if record.confirmed is False and record not in self.unconfirmed_txs:
+                self.unconfirmed_txs.append(get_light_transaction_record(record))
```

Comment:
is it possible for an attacker to send coin updates that a victim wallet will keep adding to this list? until running out of RAM?